### ISSUE TITLE
Suggested hparams for video diffusion

### ISSUE THREAD

**Issue by ezhang7423** [lucidrains/imagen-pytorch#286]
Hi there! I'm attempting to train a text to video diffusion model  for videos of size 200x200 with 64 frames. Does anyone have experience with training video diffusion models and have advice on suggested hparams, especially batch size and learning rate? Also curious what other hparams are critical for good performance here.
